Makefile

```diff
@@ -88,11 +110,7 @@ test-coverage:
 		./...
 
 test-lint:
-	golangci-lint run
-
-lintdeps-install:
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | \
-		sh -s -- -b $(shell go env GOPATH)/bin $(GOLANGCI_LINT_VERSION)
+	$(CACHE_BIN)/golangci-lint run
```

Comment:
`$(LINT)`?

Reply:
`$(LINT)` is `$(GOLANGCI_LINT) run ./... --disable-all --deadline=5m --enable

`$(GOLANGCI_LINT) run` ?

Reply:
yeah, is there a reason why we need both?  can it be `$(LINT)` here?

Reply:
oh i see, `$(LINT)` needs a sublinter name.  if `$(LINT) all` doesn't work, forget it; never mind.

Reply:
looks like `$(LINT)` is not right name to it.
maybe `SUBLINT`